theme/n21/style.scss

```diff
@@ -10,6 +10,7 @@ $colorGrey: rgba(232, 232, 232, 0.5);
 $colorLightGrey: rgba(0,0,0,0.1);
 $colorDarkerGrey: #555555;
 $colorWhite: #fff;
+$colorSemiTransparent: rgba(0,0,0,0.45);
```

Comment:
Kann nach merge von https://github.com/schul-cloud/schulcloud-client/pull/1250 wieder raus, wurde in die lib/colors.scss umgezogen